dolweb/downloads/templates/downloads-index.html

```diff
@@ -30,7 +30,7 @@ <h1>{% trans "Releases" %}</h1>
 {% endblocktrans %}</div>
 
 <div class="alert alert-danger">
-    <p>{% blocktrans %}The Windows releases <b>require</b> the <a href="https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads">64-bit Visual C++ redistributable for Visual Studio 2022</a> to be installed.{% endblocktrans %}</p>
+    <p>{% blocktrans %}The Windows releases <b>require</b> the <a href="https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170">64-bit Visual C++ redistributable for Visual Studio 2022</a> to be installed.{% endblocktrans %}</p>
```

Comment:
The new URL is what the old one had been redirecting to, so the change as it is is fine. Though it would be nice to include the fragment to reduce the need to scroll down.
```suggestion
    <p>{% blocktrans %}The Windows releases <b>require</b> the <a href="https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#latest-microsoft-visual-c-redistributable-version">64-bit Visual C++ redistributable for Visual Studio 2022</a> to be installed.{% endblocktrans %}</p>
```

Reply:
IMO we should link to the proper support page, not directly to the binary. That direct URL is not stable either, as Microsoft has changed it already a few times in the past...

Reply:
Yeah, I think the support page would be a more stable link.